core/imt/src/TImplicitMT.cxx

```diff
@@ -29,6 +30,19 @@ static std::shared_ptr<ROOT::Internal::TPoolManager> &R__GetPoolManagerMT()
    return schedMT;
 }
 
+UInt_t &R__GetIMTPoolSize()
+{
+   static UInt_t poolSize;
+   return poolSize;
+}
+
+static std::shared_ptr<tbb::task_arena> &R__GetIMTTaskArena()
```

Comment:
If this is local to the file and static, what's the problem with just declaring a `static tbb::task_arena gIMTArena;` at the top of the file?

Reply:
The idea was this not to be static for the file with further upcoming changes... meaning It slipped into the wrong commit. Will remove as it is now.